### PR TITLE
Added stutter testsuite Test[V2]

### PR DESCRIPTION
**Pull request by PraveenPenguin**
stutter testsuite which run memory Test(memhog)

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
job log : https://paste.fedoraproject.org/440258/51510147/

File: memory/stutter.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                     "master.zip"]
+        tarball = self.fetch_asset("stutter.zip", locations=locations,
+                                   expire='7d')
+        archive.extract(tarball, self.srcdir)
+        self.srcdir = os.path.join(self.srcdir, 'stutter-master')
+
+        mem_byte = str(memory.memtotal())
+        self._memory = self.params.get('memory', default=mem_byte)
+        self._iteration = self.params.get('iteration', default='10')
+        self._logdir = self.params.get('logdir', default='/root/logdir')
```
Comment (apahim) [inline memory/stutter.py]: Using `/root` here will make the test not usable by regular users.

File: memory/stutter.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        self._rundir = self.params.get('rundir', default='/tmp')
+
+        process.run('mkdir -p  %s' % self._logdir)
+
+        # export env variable, used by test script
+        os.putenv('MEMTOTAL_BYTES', self._memory)
```
Comment (apahim) [inline memory/stutter.py]: `os.putenv()` will not update `os.environ`. Not a problem here, but it is a good practice to use `os.environ['MEMTOTAL_BYTES'] = self._memory` instead.

File: memory/stutter.py (hunk 1 above)
Comment (PraveenPenguin) [inline memory/stutter.py]: @apahim  thanks , sure that make sense

File: memory/stutter.py (hunk 2 above)
Comment (PraveenPenguin) [inline memory/stutter.py]: sure

**Comment by PraveenPenguin** [comment]
@apahim raised New PR #79  with suggested change Please have a look .

Regards
Praveen

**Pull request closed without merging**